pkg/util/util.go

```diff
@@ -64,6 +64,10 @@ func IsRunModeLocal() bool {
 
 // GetOperatorNamespace returns the namespace the operator should be running in.
 var GetOperatorNamespace = func(logger logr.Logger) (string, error) {
+	ns, found := os.LookupEnv(OperatorNamespaceEnv)
+	if found {
+		return ns, nil
+	}
```

Comment:
Yes, and that's what I would like to understand better, because if it's running locally (referring to the `main` branch), it returns empty value for namespace and an error. Now, I'm not sure what exactly the behaviour should be. I was inclined to do `os.LookupEnv(OperatorNamespaceEnv)` inside the `if IsRunModeLocal()` instead of returning an empty string and an error. What do you recommend, or what would be the right thing to 
do?

```go
if IsRunModeLocal() {
	ns, found := os.LookupEnv(OperatorNamespaceEnv)
	if found {
		return ns, nil
	}
	return "", ErrRunLocal
}
```